src/main/java/picard/util/IntervalListTools.java

```diff
@@ -13,14 +37,16 @@
 import picard.cmdline.CommandLineProgram;
 import picard.cmdline.programgroups.IntervalsManipulationProgramGroup;
 import picard.cmdline.StandardOptionDefinitions;
+import picard.util.IntervalList.IntervalListScatterMode;
+import picard.util.IntervalList.IntervalListScatterer;
 
 import java.io.File;
 import java.text.DecimalFormat;
 import java.util.*;
 
 /**
  * Performs various {@link IntervalList} manipulations.
- *
+ * <p>
```

Comment:
You don't need `<p>` before any `<hX>` tag, unless you want an extra blank line. There should already be enough vertical spacing. You can use control-J in the IDE to see what it will look like. Unless you're looking at the html output and it needs this to look OK?

Reply:
the ide added it automagically for me when I asked it to rearrange the lines...

Reply:
I wonder what settings you're using that's causing that behavior.. when I try reformatting the copy in /master, the only changes I get are adding `<p>` on lines 43 and 54, which are the only places where it's actually needed. Maybe this is a bug in the IDE that's fixed in a newer version?